x/pool-incentives/keeper/keeper.go

```diff
@@ -217,13 +217,30 @@ func (k Keeper) GetAllGauges(ctx sdk.Context) []incentivestypes.Gauge {
 	return gauges
 }
 
+// IsPoolIncentivized returns a boolean representing whether the given pool ID
+// corresponds to an incentivized pool. It fails quietly by returning false if
+// the pool does not exist or does not have any records, as this is technically
+// equivalent to the pool not being incentivized.
 func (k Keeper) IsPoolIncentivized(ctx sdk.Context, poolId uint64) bool {
-	lockableDurations := k.GetLockableDurations(ctx)
+	pool, err := k.poolmanagerKeeper.GetPool(ctx, poolId)
+	if err != nil {
+		return false
+	}
+	isCLPool := pool.GetType() == poolmanagertypes.Concentrated
+
+	var gaugeDurations []time.Duration
+	if isCLPool {
+		incParams := k.incentivesKeeper.GetEpochInfo(ctx)
+		gaugeDurations = []time.Duration{incParams.Duration}
+	} else {
+		gaugeDurations = k.GetLockableDurations(ctx)
+	}
+
 	distrInfo := k.GetDistrInfo(ctx)
 
 	candidateGaugeIds := []uint64{}
-	for _, lockableDuration := range lockableDurations {
-		gaugeId, err := k.GetPoolGaugeId(ctx, poolId, lockableDuration)
+	for _, gaugeDuration := range gaugeDurations {
+		gaugeId, err := k.GetPoolGaugeId(ctx, poolId, gaugeDuration)
```

Comment:
what was the reason for changing the verbiage from lockableDurations to gaugeDurations?

Reply:
i think lockableDurations makes sense